src/NeutralLandscapes.jl

```diff
@@ -3,56 +3,41 @@ module NeutralLandscapes
 import NaNMath
 using StatsBase: sample
 using Random: rand!
-using Statistics: quantile
-using Distributions: Normal
 using Statistics: quantile, mean
+using Distributions: Normal
 using NearestNeighbors: KDTree, knn, nn
 using DataStructures: IntDisjointSets, union!, find_root, push!
+using Base: @kwdef
```

Comment:
As far as I can tell it's only used by a single type - can't we just remove it?

Reply:
`@kwdef` is used on all types now

Reply:
:+1: let's keep it